lib/rules/web/admin_console/4.5/operator_hub.xyaml

```diff
@@ -353,12 +353,12 @@ wait_filter_section_loaded:
 choose_one_checkbox:
   element:
     selector:
-      xpath: //input[contains(@class, 'pf-c-check__input') and @title='<checkbox_text>']
+      xpath: //input[contains(@class, 'pf-c-check__input') and contains(@title,'<checkbox_text>')]
     op: click
 check_label_and_value_in_modal:
   element:
     selector:
-      xpath: //h5[contains(@class,'properties-side-panel-pf-property-label') and text()='<label>']/../div[contains(@class,'properties-side-panel-pf-property-value')]/div[text()='<value>']
+      xpath: //h5[contains(@class,'properties-side-panel-pf-property-label') and text()='<label>']/../div[contains(@class,'properties-side-panel-pf-property-value')]/div[contains(text(),'<value>')]
```

Comment:
The xpath looks a bit complicated, but it should works. Can it be shortened?

Reply:
I think so, good idea @JianLi-RH  let me update